### PR TITLE
Feature/20 update base markup with bootstrap

### PR DESCRIPTION
**Pull request by ander21**

### CODE REVIEW

File: BobManager.Client/client/src/assets/style/theme.css [hunk 1]
```diff
@@ -955,6 +994,16 @@ section {
     }
 }
 
+.card-body{
+    background-color: #3755A6;
+}
+
```
Comment (viviviviviivilll) [inline BobManager.Client/client/src/assets/style/theme.css:1000]: Remove empty line

**Review by viviviviviivilll — APPROVED** [review]
Approve

File: BobManager.Client/client/src/assets/style/theme.css [hunk 2]
```diff
@@ -39,6 +43,27 @@ p {
...
+.card-title{
+    font-size: 26px;
+}
+
+.card-text{
+    font-size: 17px;
+}
+
```
Comment (NerevitVasyan) [inline BobManager.Client/client/src/assets/style/theme.css]: format with space like .container { }

File: BobManager.Client/client/src/app/home/home.component.html [hunk 3]
```diff
@@ -1,3 +1,63 @@
-<p>
-  home works!
-</p>
+<div class="container">
+  <div class="content">
+    <div class="logo">
+      <a href="#">
+        <img src="assets/png/main-logo.png" />
```
Comment (Liosh-a) [inline BobManager.Client/client/src/app/home/home.component.html]: remove empty space

File: BobManager.Client/client/src/app/home/home.component.html [hunk 4]
```diff
@@ -1,3 +1,63 @@
...
+      </a>
+    </div>
+    <div class="row m-l-10 m-r-10">
+      <div class="col-xl-4 col-md-6 col-sm-12 m-t-30">
+        <div class="card">
+          <div class="card-header text-center">
+            <strong class="card-title">Task Management</strong>
+          </div>
+          <div class="card-body">
+            <img class="rounded-circle mx-auto d-block card-img" src="assets/png/home/tasks2PNG.png" alt="Tasks" />
```
Comment (Liosh-a) [inline BobManager.Client/client/src/app/home/home.component.html]: remove empty space

File: BobManager.Client/client/src/app/home/home.component.html [hunk 5]
```diff
@@ -1,3 +1,63 @@
...
+        </div>
+      </div>
+      <div class="col-xl-4 col-md-6 col-sm-12 m-t-30">
+        <div class="card">
+          <div class="card-header text-center">
+            <strong class="card-title">Calendar</strong>
+          </div>
+          <div class="card-body">
+            <img class="rounded-circle mx-auto d-block card-img" src="assets/png/home/calendar2PNG.png"
+              alt="calendar" />
```
Comment (Liosh-a) [inline BobManager.Client/client/src/app/home/home.component.html]: remove empty space

File: BobManager.Client/client/src/app/home/home.component.html [hunk 6]
```diff
@@ -1,3 +1,63 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="col-xl-4 col-md-6 col-sm-12 m-t-30">
+        <div class="card">
+          <div class="card-header text-center">
+            <strong class="card-title">Wallet</strong>
+          </div>
+          <div class="card-body">
+            <img class="rounded-circle mx-auto d-block card-img" src="assets/png/home/wallet2PNG.png" alt="wallet" />
```
Comment (Liosh-a) [inline BobManager.Client/client/src/app/home/home.component.html]: remove empty space

File: BobManager.Client/client/src/app/consts/const.ts [hunk 7]
```diff
@@ -1 +1 @@
-export const baseAPI :string = 'http://loacalhost:00000' 
\ No newline at end of file
+export const baseAPI :string = 'http://localhost:00000'
```
Comment (Liosh-a) [inline BobManager.Client/client/src/app/consts/const.ts]: incorrect api

**Review by Liosh-a — APPROVED** [review]
approve

**Review by NerevitVasyan — APPROVED** [review]
АПРУВ

**Pull request merged**